recipe/magento2.php

```diff
@@ -171,8 +177,6 @@
 
 desc('Magento2 deployment operations');
 task('deploy:magento', [
-    'magento:compile',
-    'magento:deploy:assets',
```

Comment:
What about separating it into "build" task what is enabled by default and add doc on how to disable this step?

Reply:
Ok I did just that.

Now we can disable 'pull' strategy by adding only 3 lines;

```
task('deploy:vendors')->disabled();
task('deploy:update_code')->disabled();
task('build')->disabled();
```